styles/Board.module.css

```diff
@@ -0,0 +1,116 @@
+.container {
+    padding-top: calc(0 * var(--spacing-unit));
+}
+
+.title {
+    margin-bottom: calc(5.5 * var(--spacing-unit));
+    text-align: center;
+    color: var(--primary-blue);
+}
+
+.board_box {
+    display: flex;
+    flex-direction: row;
+    flex-wrap: wrap;
+    justify-content: center;
+    background-color: var(--primary-blue);
+    padding: var(--spacing-unit);
+    gap: calc(1.75 * var(--spacing-unit));
+}
+
+.card_box {
+    width: calc(20.62 * var(--spacing-unit));
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: flex-start;
+    text-align: justify;
+    color: var(--button-bg-outline-disabled);
+    gap: calc(3.125 * var(--spacing-unit));
+}
+
+.image_box {
+    width: auto;
```

Comment:
Please use squared images and use CSS to get it rounded

Reply:
I didn't find the original files